src/Kmd.Logic.Audit.Client.AzureBlobOrEventHubSink/AzureBlobServiceHelper.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.IO;
+using Serilog.Events;
+using Serilog.Formatting;
+
+namespace Kmd.Logic.Audit.Client.AzureBlobOrEventHubSink
+{
+    public static class AzureBlobServiceHelper
+    {
+        public static string PrepareBlobContentForUpload(ITextFormatter textFormatter, LogEvent logEvent)
+        {
+            if (textFormatter == null)
+            {
+                throw new ArgumentNullException(nameof(textFormatter));
+            }
+
+            if (logEvent == null)
+            {
+                throw new ArgumentNullException(nameof(logEvent));
+            }
+
+            string content;
+
+            using (var tempStringWriter = new StringWriter())
```

Comment:
is it possible to change the using statement to using declaration. We can do it for all files where ever possible

Reply:
This feature is not supported in C# version 7.3. Will do it once the project is updated.